tests/test_secrets.py

```diff
@@ -89,15 +96,13 @@ def register_secrets(self):
 @pytest.fixture
 def ds():
     return Datasette(
-        config={
-            "plugins": {
-                "datasette-secrets": {
-                    "database": "_internal",
-                    "encryption-key": TEST_ENCRYPTION_KEY,
-                }
-            },
-            "permissions": {"manage-secrets": {"id": "admin"}},
-        }
+        plugin_config={
+            "datasette-secrets": {
+                "database": "_internal",
+                "encryption-key": TEST_ENCRYPTION_KEY,
+            }
+        },
+        permissions={"manage-secrets": {"id": "admin"}},
```

Comment:
Using this new feature in `datasette-test`:
- https://github.com/datasette/datasette-test/issues/4